docs/basics/server.md

```diff
@@ -514,7 +514,7 @@ available to scripts provided to Gremlin Server - `graph` and `g`.
 At this point, Gremlin Server is configured and can be used to connect
 to a new or existing JanusGraph database. To start the server:
 ```bash
-$ bin/gremlin-server.sh conf/gremlin-server-janusgraph.yaml
+$ bin/gremlin-server.sh console conf/gremlin-server-janusgraph..yaml
```

Comment:
Remove additional dot. I.e. from `gremlin-server-janusgraph..yaml` to `gremlin-server-janusgraph.yaml`.
Also, `gremlin-server.sh` doesn't accept `console` parameter. I guess we need to change it to `janusgraph-server.sh`

Reply:
I revert this change. I already created a ticket for this section, see issue #2146.